nwbinspector/checks/tables.py

```diff
@@ -79,7 +80,11 @@ def check_time_intervals_stop_after_start(time_intervals: TimeIntervals, nelems:
         very long so you don't need to load the entire array into memory. Use None to
         load the entire arrays.
     """
-    if np.any(np.asarray(time_intervals["stop_time"][:nelems]) - np.asarray(time_intervals["start_time"][:nelems]) < 0):
+    if np.any(
+        np.asarray(_subsample_data(data=time_intervals["stop_time"], nelems=nelems))
+        - np.asarray(_subsample_data(data=time_intervals["start_time"], nelems=nelems))
```

Comment:
```suggestion
        _subsample_data(data=time_intervals["stop_time"], nelems=nelems)
        - _subsample_data(data=time_intervals["start_time"], nelems=nelems)
```

do we really need the np.array wrappers?